packages/wrangler/src/versions/rollback/index.ts

```diff
@@ -104,13 +108,45 @@ export async function versionsRollbackHandler(args: VersionsRollbackArgs) {
 		return;
 	}
 
-	await spinnerWhile({
-		async promise() {
-			await createDeployment(accountId, workerName, rollbackTraffic, message);
-		},
-		startMessage: `Performing rollback`,
-		endMessage: "",
-	});
+	logger.log("Performing rollback...");
```

Comment:
I think this is not a great experience. It breaks the layout of the output.

Before:

```
? Are you sure you want to deploy this Worker Version to 100% of traffic?
🤖 Using fallback value in non-interactive context: yes
├ Performing rollback
│
```

After:

```
? Are you sure you want to deploy this Worker Version to 100% of traffic?
🤖 Using fallback value in non-interactive context: yes
Performing rollback
│
```